src/Integration/LocalServices/ErrorListInfoBarController.cs

```diff
@@ -170,13 +175,22 @@ private void CancelQualityProfileProcessing()
             this.CurrentBackgroundProcessor = null;
         }
 
-        internal /*for testing purposes*/ void ProcessSolutionBinding()
+        private void ProcessSolutionBinding()
+        {
+            logger.LogDebug("[ErrorListInfoBarController] Processing solution binding...");
+            ProcessSolutionBindingAsync().Forget();
+        }
+
+        internal/* for testing */ async Task ProcessSolutionBindingAsync()
         {
+            logger.LogDebug("[ErrorListInfoBarController] Processing solution binding...");
+
             ETW.CodeMarkers.Instance.ErrorListControllerProcessStart();
 
-            // We could be on a UI thread so any unhandled exceptions will crash VS.
```

Comment:
This comment is no longer accurate - we're shouldn't be on the UI thread. However, we're still handling exceptions.